### ISSUE TITLE
Insufficient contrast on emphasized text in important notice block

### ISSUE THREAD

**Issue by wotnak** [spineldev/website#1]
Currently, color contrast (text to background) of emphasized text in important notice block is 3.53.  
It should be at least 4.5 (wcag aa).
 
We should make the background a bit of brighter and text color bit darker. We can use text color suggested in chrome devtools.
